css/style.css

```diff
@@ -0,0 +1,683 @@
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+html {
+  font-size: 15px;
+}
+
+body {
+  padding-left: 10px;
+  padding-right: 10px;
+  background-color: #f4f5f6;
+}
+
+.header {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr;
+  grid-template-rows: 2fr 1fr;
+  margin-bottom: 1px;
+  height: 150px;
+}
+
+.top-header {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr;
+  grid-template-rows: 2fr 1fr;
+  grid-column: 1 / 4;
+  grid-row: 1 / 3;
+  height: 110px;
+  background-color: #ef4430;
+  clip-path: polygon(0% 0%, 100% 0%, 100% 100%, 95% 100%, 94% 88%, 0% 88%);
+}
+
+.top-story-1 {
+  display: none;
+}
+
+.first {
+  display: flex;
+  align-items: center;
+  grid-column: 1 / 2;
+  grid-row: 1 / 3;
+}
+
+.second {
+  grid-column: 2 / 3;
+  grid-row: 1 / 3;
+}
+
+.third {
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-template-rows: 1fr;
+  grid-column: 3 / 4;
+  grid-row: 1 / 3;
+}
+
+.third-cont {
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+  padding-right: 30px;
+}
+
+.first-cont {
+  padding-left: 30px;
+}
+
+.bottom-header {
+  grid-column: 1 / 4;
+  grid-row: 3 / 4;
+  height: 40px;
+}
+
+.icon-header,
+.icon-header-1 {
+  padding-bottom: 3px;
+  color: white;
+  font-size: 900;
+}
+
+.icon-burger {
+  display: none;
+}
+
+.icon-header:hover {
+  text-decoration: none;
+}
+
+.logo-1 {
+  width: auto;
+}
+
+.mobile {
+  display: none;
+}
+
+.container-fluid-1 {
+  background-color: white;
+}
+
+.logo {
+  height: 30px;
+  width: 30px;
+}
+
+.container-fluid {
+  background-color: white;
+  margin: 0;
+  padding-right: 30px;
+  padding-left: 30px;
+}
+
+.content-1 {
+  font-size: 15px;
+  line-height: 20px;
+  font-weight: 400;
+}
+
+.top-story {
+  height: 1810px;
+  overflow-y: scroll;
+}
+
+.card-text {
+  display: flex;
```

Comment:
This is another opportunity to use bootstrap classes. ([Flex](https://getbootstrap.com/docs/4.3/utilities/flex/))